commands/init.lua

```diff
@@ -119,8 +119,14 @@ return function ()
   local message = "Enter to continue"
   local finish = prompt("Is this ok?", message)
 
-  if finish == message then
-    local data, err = fs.writeFile(cwd .. "/" .. output, data)
+  local output_path = cwd .. "/" .. output
```

Comment:
This should probably be using pathjoin. Not sure why it wasn't.

Reply:
I've noticed this as well but looking at the context I can't think of a case where it would break. `output` is always a string defined by us and `cwd` is always an absolute path

Reply:
Added it just in case, future-proofing can't hurt.